packages/devtools_app/lib/src/screens/network/har_network_data.dart

```diff
@@ -147,28 +152,25 @@ class HarDataEntry {
   }
 
   // Convert list of headers to map
-  static void _convertHeaders(Map<String, dynamic> requestData) {
+  static void _convertHeaders(Map<String, Object?> requestData) {
+    final reqData =
+        requestData[NetworkEventKeys.request.name] as Map<String, Object?>;
     // Request Headers
-    if (requestData['request'] != null &&
-        (requestData['request'] as Map<String, Object?>)['headers'] != null) {
-      if ((requestData['request'] as Map<String, Object?>)['headers'] is List) {
-        (requestData['request'] as Map<String, Object?>)['headers'] =
-            _convertHeadersListToMap(
-          ((requestData['request'] as Map<String, Object?>)['headers'])
-              as List<dynamic>,
+    if (reqData[NetworkEventKeys.headers.name] != null) {
```

Comment:
you can remove this check. If `reqData[NetworkEventKeys.headers.name]` is null, the next if check will not be true (`reqData[NetworkEventKeys.headers.name] is List`). Same for `resData[NetworkEventKeys.headers.name] != null` below

Reply:
done